coaster/gfm.py

```diff
@@ -37,15 +38,20 @@
 GFM_TAGS['thead'] = ['align', 'char', 'charoff', 'valign']
 GFM_TAGS['tr'] = ['align', 'char', 'charoff', 'valign']
 
-markdown_convert_text = Markdown(safe_mode='escape', output_format='html5',
-    enable_attributes=False,
-    extensions=['markdown.extensions.codehilite', 'markdown.extensions.smarty'],
+
+class EscapeHtml(Extension):
+    def extendMarkdown(self, md):
+        md.preprocessors.deregister('html_block')
+        md.inlinePatterns.deregister('html')
+
+
+markdown_convert_text = Markdown(output_format='html5', enable_attributes=False,
```

Comment:
https://github.com/Python-Markdown/markdown/blob/f51125d01b88067d8523e9706cfa4558b3808222/docs/change_log/release-3.0.md#output_formats-simplified-to-html-and-xhtml

Reply:
Also remove `enable_attributes` since it's no longer processed.